styles.css

```
@@ -0,0 +1,340 @@
+/* ====================== UTILITY STYLES ========================== */
+
+
+.u-button-reset {
+    background: transparent;
+    border: 0;
+    cursor: pointer;
+    font-family: inherit;
+    font-size: inherit;
+    font-weight: inherit;
+    padding: 0;
+    text-align: inherit;
+}
+
+.u-list-reset {
+    list-style: none;
+    margin-top: 0;
+    margin-bottom: 0;
+    padding-left: 0;
+}
+
+
+
+
+/* ====================== BASE STYLES ========================== */
+
+
+* {
+    box-sizing: border-box;
+}
+
+body {
+    color: black;
+    font-family: 'PT Sans', sans-serif;
+    font-size: 16px;
+    line-height: 20px;
+    margin: 0;
+}
+
+strong {
+    color: rgb(243, 123, 120);
+}
+
+a {
+    color: white;
+    text-decoration: none;
+}
+
+a:hover,
+a:focus {
+    color: black;
+}
+
+header {
+    background: linear-gradient(to right, rgb(137, 192, 115), rgb(243, 123, 120));
+    padding: 15px;
+    position: sticky;
+    top: 0;
+}
+
+img {
+    max-width: 100%;
+}
+
+h1 {
+    color: rgb(137, 192, 115);
+    font-family: 'Sansita Swashed', cursive;
+    font-size: 28px;
+    line-height: 32px;
+    text-align: center;
+}
+
+h2 {
+    color: rgb(137, 192, 115);
+    font-family: 'Sansita Swashed', cursive;
+    font-size: 25px;
+    margin-bottom: 50px;
+    margin-top: 60px;
+    text-align: center;
+}
+
+input {
+    border: 1px solid rgb(243, 123, 120);
+    border-radius: 4px;
+    height: 25px;
+    width: 60%;
+}
+
+input:hover,
+input:focus {
+    background-color: rgb(255, 232, 231);
+    outline: none;
+}
+
+footer {
+    background: rgb(137, 192, 115);
+    margin-top: 60px;
+    padding: 40px;
+    line-height: 32px;
+    text-align: center;
+    text-transform: uppercase;
+}
+
+
+
+
+/* ====================== CLASS STYLES ========================== */
+
+
+.menu-icon__line {
+    background-color: white;
+    height: 2px;
+    margin: 8px auto;
+    transition: all 260ms ease-in-out;
+    width: 30px;
+}
+
+.menu-list a {
+    display: block;
+    font-family: 'Sansita Swashed', cursive;
+    font-size: 26px;
+    padding: 40px;
+    text-align: center;
+}
+
+.details {
+    text-align: left;
+    margin-left: 20px;
+    margin-right: 20px;
+}
+
+.speakers {
+    background: rgb(243, 123, 120);
+}
+
+.speakerheading {
+    color: white;
+    padding-top: 40px;
+}
+
+.speaker-info {
+    background: white;
+    border-radius: 10px;
+    margin-bottom: 40px;
+    margin-left: 20px;
+    margin-right: 20px;
+    padding: 20px;
+}
+
+.title {
+    color: rgb(137, 192, 115);
+    font-size: 25px;
+    font-weight: bold;
+    line-height: 27px;
+    margin-bottom: -5px;
+    margin-top: 0px;
```

When writing values of zero, you can [leave off units of measurement](https://mica-web.github.io/learn/css/best-practices#avoid-using-units-on-measurements-of-zero).
```suggestion
    margin-top: 0;
```